doxygen-awesome.css

```
@@ -131,16 +131,17 @@ SOFTWARE.
 
     /* fragment colors */
     --fragment-background: #282c34;
-    --fragment-foreground: #fff;
+    --fragment-foreground: #ffffff;
     --fragment-keyword: #cc99cd;
     --fragment-keywordtype: #ab99cd;
     --fragment-keywordflow: #e08000;
     --fragment-token: #7ec699;
-    --fragment-comment: #999;
+    --fragment-comment: #999999;
     --fragment-link: #98c0e3;
     --fragment-preprocessor: #65cabe;
-    --fragment-linenumber-color: #ccc;
+    --fragment-linenumber-color: #cccccc;
     --fragment-linenumber-background: #35393c;
+    --fragment-linenumber-border: #454545;
```

```suggestion
    --fragment-linenumber-border: #1f1f1f;
```

this is closer to the current default theme border color.

---

Yeah that works better for the default theme.